src/targetPacks/ILsrc/microsoft.netcore.app/2.1.17/microsoft.netcore.app.nuspec

```diff
@@ -0,0 +1,121 @@
+        <dependency id="Microsoft.VisualBasic" version="10.1.0" />
+        <dependency id="NETStandard.Library" version="1.6.1" />
+        <dependency id="runtime.native.System.Security.Cryptography.OpenSsl" version="4.3.1" />
+        <dependency id="System.Buffers" version="4.3.0" />
+        <dependency id="System.Collections.Immutable" version="1.3.0" />
+        <dependency id="System.ComponentModel" version="4.3.0" />
+        <dependency id="System.ComponentModel.Annotations" version="4.3.0" />
+        <dependency id="System.Diagnostics.DiagnosticSource" version="4.3.1" />
+        <dependency id="System.Diagnostics.Process" version="4.3.0" />
+        <dependency id="System.Dynamic.Runtime" version="4.3.0" />
+        <dependency id="System.Globalization.Extensions" version="4.3.0" />
+        <dependency id="System.IO.FileSystem.Watcher" version="4.3.0" />
+        <dependency id="System.IO.MemoryMappedFiles" version="4.3.0" />
+        <dependency id="System.IO.UnmanagedMemoryStream" version="4.3.0" />
+        <dependency id="System.Linq.Expressions" version="4.3.0" />
+        <dependency id="System.Linq.Parallel" version="4.3.0" />
+        <dependency id="System.Linq.Queryable" version="4.3.0" />
+        <dependency id="System.Net.Http" version="4.3.2" />
+        <dependency id="System.Net.NameResolution" version="4.3.0" />
+        <dependency id="System.Net.Requests" version="4.3.0" />
+        <dependency id="System.Net.Security" version="4.3.1" />
+        <dependency id="System.Net.WebHeaderCollection" version="4.3.0" />
+        <dependency id="System.Numerics.Vectors" version="4.3.0" />
+        <dependency id="System.Reflection.DispatchProxy" version="4.3.0" />
+        <dependency id="System.Reflection.Metadata" version="1.4.1" />
+        <dependency id="System.Reflection.TypeExtensions" version="4.3.0" />
+        <dependency id="System.Resources.Reader" version="4.3.0" />
+        <dependency id="System.Runtime.Loader" version="4.3.0" exclude="Compile" />
+        <dependency id="System.Security.Cryptography.Algorithms" version="4.3.0" />
+        <dependency id="System.Security.Cryptography.Encoding" version="4.3.0" />
+        <dependency id="System.Security.Cryptography.Primitives" version="4.3.0" />
+        <dependency id="System.Security.Cryptography.X509Certificates" version="4.3.0" />
+        <dependency id="System.Threading.Tasks.Dataflow" version="4.7.0" />
+        <dependency id="System.Threading.Tasks.Extensions" version="4.3.0" />
+        <dependency id="System.Threading.Tasks.Parallel" version="4.3.0" />
+        <dependency id="System.Threading.Thread" version="4.3.0" />
+        <dependency id="System.Threading.ThreadPool" version="4.3.0" />
+      </group>
+      <group targetFramework=".NETCoreApp2.0" />
+      <group targetFramework=".NETCoreApp2.1">
+        <dependency id="Microsoft.NETCore.DotNetHostPolicy" version="2.1.17" />
```

Comment:
Will this then be brought in as a prebuilt? Is it possible to trim here? +@dseefeld

Reply:
Hmm.... IDK.  I looked at the 2.1.0 version of Microsoft.NETCore.App and we didn't trim there.

Reply:
Ok, we'll probably have to also add these to "fill out" microsoft.netcore.app:

```
Microsoft.NETCore.DotNetAppHost,2.1.17
Microsoft.NETCore.DotNetHostPolicy,2.1.17
Microsoft.NETCore.DotNetHostResolver,2.1.17
```

Reply:
Added these packages.